content/tutorials/single-page-tutorials/setup_iota-js_ for_development.md

```diff
@@ -0,0 +1,190 @@
+ 
+// For better readability, some console output will be printed in a different color
+const consoleColor = '\x1b[36m%s\x1b[0m';
+async function run() {
+	// A 256-bits entropy leads to a 24 word mnemonic seed phrase
+	const mnemonic = bip39.generateMnemonic(256);
+	console.log(consoleColor, 'Copy your mnemonic seed phrase and paste it into the .env file:');
+	console.log(mnemonic, '\n');
+}
+run();
+```
+
+Run the script `create-mnemonic.js` and check the console output for your new mnemonic seed phrase:
+```console
+node create-mnemonic.js
+```
+Now you can copy the seed phrase and paste it into the previously created `.env` file.
+
+***
+
+Your `package.json` file should look something like this:
+
+```yaml
+{
+  "name": "iota-js-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@iota/iota.js": "^1.9.0-stardust.25",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+***
+
+
+## Create an address
+
+Create a new file `create-address.js` and add the following code:
+
+
+```javascript
+const { Converter } = require("@iota/util.js");
+const { Bip32Path, Bip39 } = require("@iota/crypto.js");
+const {
+  Bech32Helper,
+  Ed25519Address,
+  Ed25519Seed,
+  ED25519_ADDRESS_TYPE,
+  generateBip44Address,
+  IOTA_BIP44_BASE_PATH,
+  SingleNodeClient,
+} = require("@iota/iota.js");
+
+require("dotenv").config({ path: "./.env" });
+
+const API_ENDPOINT = "https://api.testnet.shimmer.network";
+
+async function run() {
+  const client = new SingleNodeClient(API_ENDPOINT);
+
+  const info = await client.info();
+
+  console.log("Base");
+
+  // Generate the seed from the Mnemonic
+  const baseSeed = Ed25519Seed.fromMnemonic(process.env.MNEMONIC);
+  console.log("\tSeed", Converter.bytesToHex(baseSeed.toBytes()));
+
+  // Generate the next addresses for your account.
+  console.log();
+  console.log("Generated Addresses...");
+  const addressGeneratorAccountState = {
+    accountIndex: 0,
+    addressIndex: 0,
+    isInternal: false,
+  };
+  const path = generateBip44Address(addressGeneratorAccountState);
+  const addressSeed = baseSeed.generateSeedFromPath(new Bip32Path(path));
+  const addressKeyPair = addressSeed.keyPair();
+  const indexEd25519Address = new Ed25519Address(addressKeyPair.publicKey);
+  const indexPublicKeyAddress = indexEd25519Address.toAddress();
+
+  console.log(
+    "\tAddress (Bech32)",
+    Bech32Helper.toBech32(
+      ED25519_ADDRESS_TYPE,
+      indexPublicKeyAddress,
+      info.protocol.bech32Hrp
+    )
+  );
+}
+
+run()
+  .then(() => console.log("Done"))
+  .catch((err) => console.error(err));
+
+```
+
+If you run it, it will get your mnemonic from the `.env`file, create a seed from it and generate a new address. Just run the following command and you will see your testnet shimmer address in your console!
```

Comment:
```suggestion
If you run it, it will:

1. Get your mnemonic from the `.env`file.
2. Create a seed from it.
3. Generate a new address. 

Just run the following command, and you will see your testnet shimmer address in your console!
```